robusta_krr/strategies/simple.py

```diff
@@ -38,7 +36,7 @@ def calculate_cpu_proposal(self, data: PodsTimeData) -> float:
         else:
             data_ = list(data.values())[0][:, 1]
 
-        return np.percentile(data_, self.cpu_percentile)
+        return np.max(data_)
```

Comment:
why are we switching from percentile to max here?

Reply:
Because CPU is used in the PromQL here